charts/azure/values.yaml

```diff
@@ -0,0 +1,49 @@
+nameOverride: ""
+fullnameOverride: ""
+labels: {}
+scraper:
+  name: azure
+
+subscriptionID: ''
+organisation: ''
+tenantID: ''
+
+connectionDetails:
```

Comment:
Can we add `values.schema.json` ? - I don't think these values should be empty strings rather empty and commented out.